src/clr-core/icon-shapes/icon.element.scss

```diff
@@ -132,16 +96,21 @@ svg {
 :host(.is-white),
 :host(.is-inverse),
 :host([inverse]) {
-  @include setIconColor(#{'var(--clr-icon-color-inverse, var(--clr-color-neutral-0,' + $clr-color-neutral-0 + '))'});
+  --color: #{$clr-color-neutral-0};
+}
+
+// DEPRECATED IN 3.0: is-info, is-inverse
+:host([inverse].is-info),
+:host([status='info'].is-inverse),
+:host([inverse][status='info']) {
+  --color: #{$clr-color-action-600};
```

Comment:
I wasnt sure if this was a valid state? There was no inverse info icon color so when the icon was inverse with state value info it would be white instead of blue.

Reply:
It should be a valid state, making the icon the dark theme info color for lack of a better construct...

Reply:
For reference that color would be `clr-color-action-400`